### PR TITLE
Sops3

### PR DESCRIPTION
**Pull request by jvehent**
To be merge when ready to release.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,25 @@
...
+  keys to access a data key and decrypt a file. See `sops groups -help`.
+
+* Keyservice to forward access to a local master key on a socket, similar to
+  gpg-agent. See `sops keyservice --help`.
+
+* Encrypt comments by default
+
+* Support for Google Compute Platform KMS
+
+* Refactor of the store logic to separate the internal representation SOPS
```
Comment (autrilla) [inline CHANGELOG.rst]: Do users care about this?

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,6 +1,25 @@
...
+  has of files from the external representation used in JSON and YAML files
+
+* Reencoding of versions as string on sops 1.X files, may break backward
+  compatibility but will be handled automatically.
```
Comment (autrilla) [inline CHANGELOG.rst]: It's not handled automatically, but the following error message is shown: `Error loading file metadata: Error unmarshalling input json: json: cannot unmarshal number into Go struct field Metadata.version of type string`

File: CHANGELOG.rst (hunk 1 above)
Comment (jvehent) [reply CHANGELOG.rst]: users, no, but they also don't read changelogs. developers may care, and they are the target audience.

File: CHANGELOG.rst (hunk 2 above)
Comment (jvehent) [reply CHANGELOG.rst]: What should users who see this message do?

Comment (autrilla) [reply CHANGELOG.rst]: They have two options:

- Rotate the file with any version of SOPS `2.x.x`, which will make `Metadata.version` a string.
- Manually edit the JSON file to ensure the entry under `sops -> version` is a string and not a number.

**Comment by autrilla** [comment]
Yes, I've opened #251 for it.

**Pull request merged**